_format/1.1/index.md

```diff
@@ -2144,7 +2144,8 @@ least one of:
   localization.
 * `detail`: a human-readable explanation specific to this occurrence of the
   problem. Like `title`, this field's value can be localized.
-* `source`: an object containing references to the source of the error,
+* `source`: an object containing references to the source of the error.
+  It **SHOULD** include one of the following members or be omitted:
   optionally including any of the following members:
```

Comment:
It looks like the middle line was added instead of replacing the last. I also added the word "primary" to the first line.

```suggestion
* `source`: an object containing references to the primary source of the error.
  It **SHOULD** include one of the following members or be omitted:
```

Reply:
--sigh--

I should not be allowed to wield natural languages before the first pot of coffee has soaked in.